src/commands/skip.rs

```diff
@@ -34,52 +26,14 @@ pub async fn skip(
 
     if queue.is_empty() {
         return create_response(&ctx.http, interaction, NOTHING_IS_PLAYING).await;
-    }
-
-    let mut data = ctx.data.write().await;
-    let cache_map = data.get_mut::<GuildCacheMap>().unwrap();
-
-    let cache = cache_map.entry(guild_id).or_default();
-    cache.current_skip_votes.insert(interaction.user.id);
-
-    let guild_users = ctx.cache.guild(guild_id).await.unwrap().voice_states;
-    let channel_guild_users = guild_users
-        .into_values()
-        .filter(|v| v.channel_id.unwrap() == bot_channel_id);
-    let skip_threshold = channel_guild_users.count() / 2;
-
-    if cache.current_skip_votes.len() >= skip_threshold {
+    } else {
         for _ in 1..tracks_to_skip {
             queue.dequeue(1);
         }
         if queue.skip().is_ok() {
-            create_response(&ctx.http, interaction, SKIPPED).await?
+            return create_response(&ctx.http, interaction, SKIPPED).await;
```

Comment:
I wouldn't say so because of two things.
1. It would add a bit of unnecessary complexity.
2. If the skip-to index is bigger than the queue, track number X will not exist.
However, if you agree, I can maybe do a "Skipped to {Music Name}".

Reply:
> If the skip-to index is bigger than the queue, track number X will not exist.

Take the minimum as you calculated before, no?

> However, if you agree, I can maybe do a "Skipped to {Music Name}".

I think I prefer this, yeah. But as you wish. 🙂

Reply:
Done ✅